src/xcode/ENA/ENA/Source/Developer Menu/Features/DMWarnOthersNotificationViewController.swift

```diff
@@ -58,7 +70,7 @@ final class DMWarnOthersNotificationViewController: UIViewController, UITextFiel
 		let currentSubmissionConsentStatusStateLabel = UILabel(frame: .zero)
 		currentSubmissionConsentStatusStateLabel.translatesAutoresizingMaskIntoConstraints = false
 		currentSubmissionConsentStatusStateLabel.numberOfLines = 0
-		currentSubmissionConsentStatusStateLabel.text = store.isSubmissionConsentGiven ? "🟢 Consent granted 👍" : "🔴 Consent not given 👎"
+		currentSubmissionConsentStatusStateLabel.text = ""
```

Comment:
I think this line can be removed completely.

Reply:
Sure. 👍  removed.